rllib/algorithms/ppo/ppo.py

```diff
@@ -366,11 +368,28 @@ def training_step(self) -> ResultDict:
         # workers.
         if self.workers.num_remote_workers() > 0:
             with self._timers[SYNCH_WORKER_WEIGHTS_TIMER]:
+                src_weights = None
+                if self.config._enable_rl_trainer_api:
+                    src_weights = self.trainer_runner
```

Comment:
same here.
or from_worker = None if not self.config._enable_rl_trainer_api else self.trainer_runner

Reply:
done.